pcsx2/R5900OpcodeImpl.cpp

```diff
@@ -982,8 +982,46 @@ void SYSCALL()
 		case Syscall::SetVTLBRefillHandler:
 			DevCon.Warning("A tlb refill handler is set. New handler %x", (u32*)PSM(cpuRegs.GPR.n.a1.UL[0]));
 			break;
-
-
+#ifndef USE_HARDCODED_BIOS_INFORMATION
+		case Syscall::StartThread:
+		case Syscall::ChangeThreadPriority:
+		{
+			if (CurrentBiosInformation.threadListAddr == 0)
+			{
+				u32 offset = 0x0;
+				// Suprisingly not that slow :)
+				while (offset < 0x5000) // I pulled this number out of my ass. How far _should_ we read into the kernel?
```

Comment:
Is this number accurate or not? Please try it out with more kernels to see if that works and delete that comment, or fix it otherwise :p

Reply:
I find that the instructions are in the 0x4xxxx region. I'll remove the comment